ste/sender-blockBlob.go

```diff
@@ -150,6 +150,14 @@ func (s *blockBlobSenderBase) Epilogue() {
 		ctxWithLatestServiceVersion := context.WithValue(jptm.Context(), ServiceAPIVersionOverride, azblob.ServiceVersion)
 		_, err := s.destBlockBlobURL.SetTier(ctxWithLatestServiceVersion, s.destBlobTier, azblob.LeaseAccessConditions{})
 		if err != nil {
+			if s.jptm.Info().S2SSrcBlobTier != azblob.AccessTierNone {
+				s.jptm.LogTransferInfo(pipeline.LogError, s.jptm.Info().Source, s.jptm.Info().Destination, "Failed to replicate block blob tier at destination. Try transferring with the flag --s2s-preserve-access-tier=false")
```

Comment:
Minor: maybe simply remove the word "block" from this message, since the copy may in fact be sourced from a page blob and so readers may be confused about whether "block" relates to source or dest..  (Sender-blockBlob is used for all cases where the _destination_ is a block blob)